### ISSUE TITLE
[suggestion] 'Please first open 2 documents to compare'

### ISSUE THREAD

**Issue by TuxVinyards** [ryu1kn/vscode-partial-diff#76]
Very nice diff feature with great highlighting ....  But:

Suggest wording the help to read something like:

"Please make sure you are using a split-screen & that  both files are simultaneously visible"

"Requires a split-screen &  both files to be simultaneously visible"

This would save a lot user cussing and fuming. And stop people wanting to instantly uninstall ...  Speaking personally.  LOL

(see #35)

**Comment by newmedicine** [comment]
A thousand times, yes!

**Comment by cscarp** [comment]
Yep, I was about to uninstall but checked the git issues first and found this.

**Comment by adam-gol** [comment]
Yea, even that doesn't work.

> "Please make sure you are using a split-screen & that both files are simultaneously visible"

**Comment by ryu1kn** [comment]
Thank you @TuxVinyards for the suggestion. Makes sense, fixed it in [v1.4.6](https://github.com/ryu1kn/vscode-partial-diff/releases/tag/v1.4.6).
